cockroachdb/values.yaml

```diff
@@ -426,6 +426,19 @@ init:
     #       # https://www.cockroachlabs.com/docs/stable/create-schedule-for-backup.html#schedule-options
     #       options: [first_run = 'now']
 
+copyCerts:
```

Comment:
Thank you for your contribution!
Can you edit `build/templates/values.yaml` instead of this file - the contents of this file will be replaced (it's generated).

Reply:
done

I also moved the config to the tls.copyCerts section; the image of the container is already configured there.